lighthouse/helpers/plates.py

```diff
@@ -723,3 +727,17 @@ def format_plate(barcode: str) -> Dict[str, Union[str, bool, Optional[int]]]:
         else 0,
         "count_filtered_positive": count_filtered_positive if count_filtered_positive is not None else 0,
     }
+
+    if include_samples is True:
+        formated_response["pickable_samples"] = list(map(pickable_sample_attributes, fit_to_pick_samples))
+
+    return formated_response
+
+
+def pickable_sample_attributes(sample: Any) -> Any:
```

Comment:
If possible, try to define the types more explicitly; using `Any` doesn't really give us any advantage when using type checking.